statusHandler/presenter/instanceInfoGetters.go

```diff
@@ -41,3 +51,83 @@ func (psh *PresenterStatusHandler) GetCountLeader() uint64 {
 func (psh *PresenterStatusHandler) GetCountAcceptedBlocks() uint64 {
 	return psh.getFromCacheAsUint64(core.MetricCountAcceptedBlocks)
 }
+
+// GetNodeName will return node name
```

Comment:
`GetNodeName will return the node's display name`

Reply:
not done

Reply:
done.